zproject/default_settings.py

```diff
@@ -384,6 +384,8 @@ class JwtAuthKey(TypedDict):
 
 JWT_AUTH_KEYS: Dict[str, "JwtAuthKey"] = {}
 
+JWT_FETCH_API_KEYS: Dict[str, "JwtAuthKey"] = {}
```

Comment:
Can you add comments for the two JWT settings in the style of the rest of this file here?

Reply:
Done.